test/namespace_pkgs_test.py

```diff
@@ -0,0 +1,7 @@
+from azure.mgmt import resource, compute
```

Comment:
(not usually my recommendation, but) if we're testing the imports should we move the `import` into the test case? that way if it ever fails the debugging should be a bit easier because it would happen within a test case instead of during pytest collection time?